src/web/mjs/connectors/VerMangasPorno.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from "../engine/Connector.mjs";
```

Comment:
This file is not related to a change in the frontend, i recommend to make a separate PR (furthermore i think this was already added)